src/dropzone/DropzoneDirective.ts

```diff
@@ -77,6 +77,9 @@ export class DropzoneDirective implements ng.IDirective {
           this.emit('queuecomplete');
         }
       },
+      'refresh': function(file: any) {
```

Comment:
In this case, I would move the onRefresh call to be within the 'success' event handler. The 'success' event handler defines the behavior that occurs when the success event is fired. Part of that behavior would involve the refreshing of the UI. An event handler called 'refresh' would indicate to me the behavior that is to occur once something has refreshed which is not what is happening here.